platforms/BUILD

```diff
@@ -59,3 +59,22 @@ platform(
     constraint_values = HOST_CONSTRAINTS,
     parents = [get_parent_from_constraints(HOST_CONSTRAINTS)],
 )
+
+# Target these platforms with --platforms for a static build linked against musl libc.
+platform(
```

Comment:
These platforms are really only used as target platforms, do I still need to place any `exec_properties` on them?

Reply:
Good question. I don't think there is any value in adding them today. But it could cause some confusion to why these don't come with `exec_properties`. 🤔 

If there is no way in Bazel to separate exec and target platform definitions, then I think adding some comment on top of these to explain that they should be used as target platform would suffice.